main.go

```diff
@@ -153,6 +154,7 @@ func main() {
 	flags.StringVar(&options.kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file")
 	flags.BoolVarP(&options.help, "help", "h", false, "Print help text")
 	flags.IntVar(&options.port, "port", 80, `Port to expose metrics on.`)
+	flags.IntVar(&options.ksmMetricsPort, "ksm-metrics-port", 81, `Port to expose kube-state-metrics own metrics on.`)
```

Comment:
Typically this is referred to as the "telemetry-port". And I would like to be able to specify the address/interface to bind to as well, as it may be different from `options.host`.

Reply:
Done.

Reply:
Added `--telemetry-host` and `--telemetry-host` for kube-state-metrics self metrics.